src/Database/Database.php

```diff
@@ -212,6 +212,10 @@ public function connect(array $params = null)
         $this->connection->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
         $this->connection->setAttribute(PDO::ATTR_EMULATE_PREPARES, false);
 
+        // TODO: behavior without this attribute would be preferrable
+        // (actual types instead of all strings) but would be a breaking change
+        $this->connection->setAttribute(PDO::ATTR_STRINGIFY_FETCHES, true);
```

Comment:
I think this only affects SQLite. MySQL is only affected with emulated prepared statements, which we AFAIK don't use.

Reply:
Means we need to move it elsewhere? Or just context info?

Reply:
Not sure to be honest. For MySQL the current change is a breaking change as the previous behavior was to return native types, now we always enforce strings. For SQLite the change prevents a breaking change. So if we want to avoid the breaking change for both, we should only set the attribute for SQLite connections.